students/allen_maxwell/Lesson_09/donor_models.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+
+# Allen Maxwell
+# Python 210
+# 1/20/2020
+# donor_models.py
+
+
+import os
+
+
+def check_name(name):
+    '''Checks the name entry for a Full Name (First and Last).'''
+    try:
+        name.split()[1]
+        name = name.title()
+        return name.title()
+    except TypeError:
+        raise ('Name must contain a first and last name')
+
+
+def check_donation(donation):
```

Comment:
note that these methods would be good as staticmethods

Reply:
Updated changes in Lesson 9 Assignment Update